docs/requirements.txt

```diff
@@ -0,0 +1,4 @@
+sphinx>=1.3
+sphinx-py3doc-enhanced-theme
+m2r
+-e .
```

Comment:
I wonder if these should be in a special `docs-requirements.txt` file.

Reply:
Aha!  I see this is in "docs/requirements.txt" which is OK with me.

Reply:
This is actually recommended for the readthedocs build process: https://docs.readthedocs.io/en/stable/guides/specifying-dependencies.html?highlight=requirements#specifying-a-requirements-file